byron/ledger/executable-spec/byron-spec-ledger.cabal

```diff
@@ -31,17 +34,11 @@ common project-config
                       -Wincomplete-record-updates
                       -Wincomplete-uni-patterns
                       -Wredundant-constraints
+                      -Wunused-packages
                
   if (!flag(development))
     ghc-options:      -Werror
 
-  if impl(ghc >= 8.10)
-    ghc-options:      -Wunused-packages
-
-common doctest
-  if impl(ghc < 8.10)
-    build-depends:    doctest
```

Comment:
You previously had this disabled on 8.10, but now if seems to be enabled in general

Reply:
I did it because doctests were broken.  They still are.  Trying to fix it.  🤞

Reply:
I have rebased on the other PR that runs the doctests using some other method: https://github.com/input-output-hk/cardano-ledger-specs/pull/2188

We we want to go with that then I can can delete the doctest components, since they will no longer be used anymore.